htdocs/changeapppwd.php

```diff
@@ -0,0 +1,225 @@
+<?php
```

Comment:
created a local branch from ltb-project:master and merged it locally to markus-96:1.5, then resolved the conflicts.

I have to delete my fork so that I can fork from master directly, hopefully this PR is closed as soon as possible.

Reply:
>     * adding some features (ppolicy criteria)

For entropy checks:
This can not be done between "app"-passwords and the user-password, because on the app-password side you need to know the old password, which you do not. So: While changing the "app"-password, you can check the entropy between "app"-password and user-password, but if you change the user-password, you can not check the entropy to the already set "app"-password. So this check is useless in this aspect.

I will try to adopt the added .js-functionality

Reply:
should be resolved now

Reply:
Hello,

Thanks for the taking into consideration my remarks. Globally, it is much better.

Your branch 1.5 and origin/master are in conflict, and furthermore you rebase in the wrong direction. This makes the rebase very complicated, so I preferred to do a global patch from your branch, and apply it to the master branch.

See this issue for referencing the feature: https://github.com/ltb-project/self-service-password/issues/864
And this new MR for continuing the work: https://github.com/ltb-project/self-service-password/pull/865

Reply:
Hello,

Thanks for creating the new branch. But I have some problem:

I can not push to change-custom-password-fields, how should I continue working on this? Should I create a new fork and create PRs for every commit to change-custom-password-fields?

Reply:
> Hello,
> 
> Thanks for creating the new branch. But I have some problem:
> 
> I can not push to change-custom-password-fields, how should I continue working on this? Should I create a new fork and create PRs for every commit to change-custom-password-fields?

You could clone again or synchronize your personal repository, work on your local `change-custom-password-fields` branch, and propose a PR to self-service-password `change-custom-password-fields` branch